tests/fcma/test_io.py

```diff
@@ -50,18 +57,30 @@ def test_prepare_fcma_data():
         'the labels do not match in test_prepare_fcma_data'
 
 def test_prepare_mvpa_data():
-    processed_data, labels = io.prepare_mvpa_data(dir, extension, mask_file, epoch_file)
+    processed_data, labels = io.prepare_mvpa_data(dir, extension, epoch_file, mask_file)
     expected_processed_data = np.load(os.path.join(os.path.dirname(__file__),
                                                    'data/expected_processed_data.npy'))
     assert len(processed_data) == len(expected_processed_data), \
         'numbers of epochs do not match in test_prepare_mvpa_data'
     for idx in range(len(processed_data)):
         assert np.allclose(processed_data[idx], expected_processed_data[idx]), \
-            'raw data do not match'
+            'raw data do not match in test_prepare_mvpa_data'
     assert np.array_equal(labels, expected_labels), \
         'the labels do not match in test_prepare_mvpa_data'
 
+def test_prepare_searchlight_mvpa_data():
+    processed_data, labels = io.prepare_searchlight_mvpa_data(dir, extension, epoch_file)
+    expected_searchlight_processed_data = \
+        np.load(os.path.join(os.path.dirname(__file__),
+                             'data/expected_searchlight_processed_data.npy'))
```

Comment:
If you use a `/` in your path string, there is little benefit from using `os.path.join`.